client/landing/stepper/declarative-flow/ai-assembler.ts

```diff
@@ -29,6 +29,7 @@ const SiteIntent = Onboard.SiteIntent;
 
 const withAIAssemblerFlow: Flow = {
 	name: AI_ASSEMBLER_FLOW,
+	isSignupFlow: true,
```

Comment:
Considering this flag is very important, what do you think about updating the type to make it required?

So the flow that the user signup is not required should explicitly set is as false.

Reply:
That's a great point! No strong preference here, my two cent is that I would delegate the responsibility of adding the flag to the person that's implementing the new flow. While the flag is important, it's not essential for the flow to function.

Reply:
I have mixed feelings. My first thought is that I agree with @taipeicoder. But all of the existing flows are currently signup flows so it may make more sense to go with @gabrielcaires 's suggestion.